src/UtilityRepository.js

```diff
@@ -0,0 +1,42 @@
+class UtilityRepository {
+  constructor(userData, sleepData, hydrationData, activityData) {
+    this.userData = userData;
+    this.sleepData = sleepData;
+    this.hydrationData = hydrationData;
+    this.activityData = activityData;
```

Comment:
Look into `Object.assign()` for making this logic a one-liner. You'd have to pass in an object of all that data like:

```js
constructor(allTotal)
```

Then use `Object.assign()` to say "assign all the values in the `allTotal` object to our `this` object. I won't show you exactly how to do this, but if you're feeling like you need a little extra push - try it out ;) Not a requirement.